Cryptomator/S3/S3LoginViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  S3LoginViewController.swift
+//  Cryptomator
+//
+//  Created by Philipp Schmid on 28.06.22.
+//  Copyright © 2022 Skymatic GmbH. All rights reserved.
+//
+
+import Combine
+import CryptomatorCommonCore
+import SwiftUI
+import UIKit
+
+class S3LoginViewController: UIHostingController<S3LoginView> {
+	weak var coordinator: (Coordinator & S3Authenticating)?
+	let viewModel: S3LoginViewModel
+	private var subscriptions = Set<AnyCancellable>()
+
+	init(viewModel: S3LoginViewModel) {
+		self.viewModel = viewModel
+		super.init(rootView: S3LoginView(viewModel: viewModel))
+	}
+
+	@available(*, unavailable)
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		let doneButton = UIBarButtonItem(barButtonSystemItem: .done, target: self, action: #selector(done))
+		navigationItem.rightBarButtonItem = doneButton
+		let cancelButton = UIBarButtonItem(barButtonSystemItem: .cancel, target: self, action: #selector(cancel))
+		navigationItem.leftBarButtonItem = cancelButton
+		title = viewModel.title
+
+		viewModel.saveDisabled.sink { disabled in
+			doneButton.isEnabled = !disabled
+		}.store(in: &subscriptions)
+
+		let hud = ProgressHUD()
+		hud.text = LocalizedString.getValue("webDAVAuthentication.progress")
```

Comment:
Value might be correct but we should use a more specific key. 😁

Reply:
Resolved by bc378db4408b121e7a9405af30e13078012dbc84.